package.json

```
@@ -34,6 +34,7 @@
     "prop-types": "^15.8.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-select": "^5.7.3",
```

```suggestion
    "react-select": "^5.5.8",
```
We must match the constraint in silverstripe/admin if we're going to add this at all.

---

We have 2 different versions now:
`"silverstripe-cms"`: `"react-select": "^5.6.0"`,
`"silverstripe-admin"`: `"react-select": "^5.5.8"`

Probably, I'll update `cms` and `admin` to `"react-select": "^5.7.3"` to be consistent.
I assume that is the best practice to explicitly require all dependencies that we use in module.
https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-extraneous-dependencies.md

---

Updated